public/app/features/alerting/unified/components/receivers/TemplatesTable.tsx

```diff
@@ -59,7 +59,7 @@ export const TemplatesTable: FC<Props> = ({ config, alertManagerName }) => {
                   <td>{name}</td>
                   <td className={tableStyles.actionsCell}>
                     <ActionIcon
```

Comment:
ActionIcon? We have a ButtonIcon component, is this similar?

Reply:
It is an `<Icon />` wrapped in `<Tooltip/>` and optionally wrapped in `<Link />`, I did not find a component to do both of these